doc/tutorials/calib3d/camera_multiview_calibration/multiview_calibration.markdown

```diff
@@ -0,0 +1,315 @@
+Multi-view Camera Calibration Tutorial {#tutorial_multiview_camera_calibration}
+==========================
+
+@tableofcontents
+
+@prev_tutorial{tutorial_interactive_calibration}
+
+|    |    |
+| -: | :- |
+| Original author | Maksym Ivashechkin |
+| Compatibility | OpenCV >= 5.0 |
+
+Structure:
+----
+This tutorial consists of the following sections:
+* Introduction
+* Briefly
+* How to run
+* Python example
+* Python visualization
+* Details Of The Algorithm
+* Method Input
+* Method Output
+* Method Input
+* Pseudocode
+* Python sample API
+* C++ sample API
+
+Introduction
+----
+Multiview calibration is a very important task in computer vision. It is widely used in 3D reconstruction, structure from motion, autonomous driving etc. The calibration procedure is often the first step for any vision task that must be done to obtain intrinsics and extrinsics parameters of the cameras. The accuracy of camera calibration parameters directly influence all further computations and results, hence, estimating precise intrinsincs / extrinsics is crucial.
+
+The calibration algorithms require a set of images for each camera, where on the images a calibration pattern (e.g., checkerboard, aruco etc) is visible and detected. Additionally, to get results with a real scale, the 3D distance between two neighbor points of the calibration pattern grid should be measured. For extrinsics calibration, images must share the calibration pattern obtained from different views, i.e., overlap of cameras' field of view. Moreover, images that share the pattern grid have to be taken at the same moment of time, or in other words, cameras must be synchronized. Otherwise, the extrinsics calibration will fail.
+
+The intrinsics calibration incorporates estimation of focal lengths, skew, and principal point of the camera; these parameters are combined in the intrinsic upper triangular matrix of size 3x3. Additionally, intrinsic calibration includes finding distortion parameters of the camera. The extrinsics parameters represent a relative rotation and translation between two cameras. Therefore, for `N` cameras, a sufficient amount of correctly selected pairs of estimated relative rotations / translations is `N-1`, while extrinsics parameters for all possible pairs ${N\choose 2} = \frac{N(N-1)}{2})$ could be derived from those that are estimated. More details about intrinsics calibration could be found in this tutorial @ref tutorial_camera_calibration_pattern, and its implementation [cv::calibrateCamera](https://docs.opencv.org/3.4/d9/d0c/group__calib3d.html#ga3207604e4b1a1758aa66acb6ed5aa65d).
```

Comment:
> https://docs.opencv.org/

Links on docs.opencv.org should be avoided.
Use `@ref` instead.

Reply:
replaced https with ref